pkg/minikube/registry/registry_test.go

```diff
@@ -39,41 +53,62 @@ func TestRegistry(t *testing.T) {
 	}
 
 	registry := createRegistry()
+	t.Run("registry.Register", func(t *testing.T) {
+		t.Run("foo", func(t *testing.T) {
+			if err := registry.Register(foo); err != nil {
+				t.Fatal("expect nil")
+			}
+		})
+		t.Run("fooAlreadyExist", func(t *testing.T) {
+			if err := registry.Register(foo); err != ErrDriverNameExist {
+				t.Fatal("expect ErrDriverNameExist")
+			}
+		})
+		t.Run("bar", func(t *testing.T) {
+			if err := registry.Register(bar); err != nil {
+				t.Fatal("expect nil")
+			}
+		})
+	})
+	t.Run("registry.List", func(t *testing.T) {
+		list := registry.List()
+		if len(list) != 2 {
```

Comment:
you are already comparing `len(list) == len(drivers)` below, no need to compare it again with 2, move the length check here.

Reply:
fixed